Formula/jetty-runner.rb

```diff
@@ -17,6 +17,8 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/jetty-runner --version", 1)
```

Comment:
Also, is there any chance of a more detailed test? We prefer not to test `version` or `help` output unless nothing else is available.